system-parachains/bridge-hubs/bridge-hub-polkadot/src/xcm_config.rs

```diff
@@ -353,3 +382,22 @@ impl cumulus_pallet_xcm::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 }
+
+pub struct XcmFeeManagerFromComponentsBridgeHub<WaivedLocations, HandleFee>(
```

Comment:
docs

Reply:
Addressed in https://github.com/polkadot-fellows/runtimes/pull/130/commits/0c6e184afa829c55e242623fb923defff61dcf2c.